Spider.php

```diff
@@ -70,6 +75,59 @@ public function getRequestHandler()
     }
 
     /**
+     * Sets the blacklist
+     *
+     * @argument array
```

Comment:
@argument isn't valid, please use `@param array $blacklist`. And please make it return it self (fluent interface).